internal/data/spicedb.go

```diff
@@ -501,6 +513,13 @@ func optionalStringToStringPointer(optional string) *string {
 	return &optional
 }
 
+func addPrefix(relation, prefix string) string {
```

Comment:
Would the name "addRelationPrefix" makes more sense? WDYT? Just to be more clear the intent of the method

Reply:
The function can take any prefix, but considering the goal is to add the prefix to a relation i can see it make sense to clarify that 👍